transaction/examples/resources/creation/non_fungible/with_initial_supply.rtm

```diff
@@ -0,0 +1,39 @@
+# This transaction manifest creates a new non-fungible resource with initial supply and with the 
+# default auth.
+
+# Locking 10 XRD in fees from the account component. 
+CALL_METHOD 
+    ComponentAddress("{account_component_address}") 
+    "lock_fee"
+    Decimal("10");
+
+# Creating a new resource 
+CREATE_RESOURCE 
+    Enum(
+        "NonFungible", 
+        Enum("U32")
+    ) 
+    Array<Tuple>(
+        Tuple("name", "MyResource"), 
+        Tuple("symbol", "RSRC"),
+        Tuple("description", "A very innovative and important resource"), 
+    ) 
+    Array<Tuple>(
+        Tuple(Enum("Withdraw"), Tuple(Enum("AllowAll"), Enum("DenyAll"))),
```

Comment:
Do you think it's worth a quick comment explaining what the two different rules are? (ie the first is for performing the action; the second is for updating either of the auth rules associated with the action)